packages/api/src/routes/releaseBadgeRoute.js

```diff
@@ -0,0 +1,25 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import express from 'express';
+
+const router = express.Router();
+import releaseBadgeController from '../controllers/releaseBadgeController.js';
+
+router.get('/', releaseBadgeController.getReleaseBadgeVersion());
+
+router.patch('/', releaseBadgeController.updateReleaseBadgeVersion());
```

Comment:
Feels weird not to have any middleware haha. Would the isSelf middleware be appropriate? Makes me thing spotlight might be missing something.

Reply:
Yup, I went through that same thinking, initially adding and then removing `isSelf`. In the end I figured maybe the rationale (for spotlight, and so for here), is that it just doesn't matter that much? I guess that would be an argument against moving this to the `user` table where a patch could have some consequence.

Reply:
Well it always matters to have good auth -- otherwise in this case any data can be written to that string column. Like a malicious html element. 

I forget if the id_token is verified against creds by this point so I am not sure its needed or anything but spotlights could just be wrong -- I also see it done on some sensor routes which also scares me haha.